loader/src/cli.js

```diff
@@ -130,23 +126,31 @@ function main() {
         Load data about COVID-19 vaccine appointment availability from a
         variety of different sources.
 
+        Data about each vaccination location is written as a single line of JSON
+        on STDOUT, so you can pipe or stream the output to other files or
+        programs for processing. Informational messages are available on STDERR.
+
```

Comment:
This isn’t anything new, but it seemed like the current behavior might be worth stating more explicitly.